contracts/src/v0.8/keystone/test/mocks/CapabilityConfigurationContract.sol

```diff
@@ -17,7 +17,7 @@ contract CapabilityConfigurationContract is ICapabilityConfiguration, ERC165 {
     s_donConfiguration[donId] = config;
   }
 
-  function supportsInterface(bytes4 interfaceId) public pure override returns (bool) {
-    return interfaceId == this.getCapabilityConfiguration.selector ^ this.beforeCapabilityConfigSet.selector;
+  function supportsInterface(bytes4 interfaceId) public pure returns (bool) {
+    return interfaceId == type(ICapabilityConfiguration).interfaceId || interfaceId == type(IERC165).interfaceId;
```

Comment:
Can you explain this change please? Why don't we need to use selectors any more?

Reply:
Combined selectors are produced by `type(ICapabilityConfiguration).interfaceId`. The main addition here is checking for `type(IERC165).interfaceId`, which was missing before.